### DISCUSSION TITLE
[Feature] Allow choosing a main image for stacks

### DISCUSSION THREAD

**Discussion by Quadrubo** [immich-app/immich#14313] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to be able to choose a main image for photo stacks.
Right now the main image always seems to be the first image of the stack.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mmomjian** [comment]
duplicate of #8498, thank you

**Reply by Quadrubo** [reply]
Ah sorry, this didn't show up when searching for "main stack" for some reason
